executor/child/withdraw.go

```diff
@@ -196,18 +203,94 @@ func (ch *Child) GetWithdrawal(sequence uint64) (executortypes.WithdrawalData, e
 	return data, err
 }
 
+func (ch *Child) GetSequencesByAddress(address string, offset uint64, limit uint64, descOrder bool) ([]uint64, error) {
+	count := uint64(0)
+	sequences := make([]uint64, 0)
+
+	fetchFn := func(key, value []byte) (bool, error) {
+		if count >= limit {
+			return true, nil
+		}
+
+		sequence, err := dbtypes.ToUint64(value)
+		if err != nil {
+			return true, err
+		}
+		sequences = append(sequences, sequence)
+		count++
+		return false, nil
+	}
+
+	if descOrder {
+		lastIndex, err := ch.GetLastAddressIndex(address)
+		if err != nil {
+			return nil, err
+		}
+		if offset > lastIndex {
```

Comment:
It can be better to use this offset as index not actual offset from the last index to prevent pagination corruption during fetch